pyproject.toml

```diff
@@ -15,11 +15,29 @@ build-backend = "setuptools.build_meta:__legacy__"
 
 
 [tool.black]
-# Uncomment if pyproject.toml worked fine to ensure consistency with flake8
-# line-length = 120
+line-length = 88
 target-version = ["py38", "py39", "py310", "py311"]
 
 
+[tool.isort]
+src_paths = ["caffe2", "torch", "torchgen", "functorch", "tests"]
```

Comment:
While we are doing changes, we should probably update lintrunner.toml's version of isort

Reply:
There is no `isort` requirement in the `lintrunner.toml`. We are using `usort` instead of `isort` as our import sorter. I added the `isort` configuration here as a placeholder if we plan to change to `ruff` in the future.

Reply:
We should also consider moving UFMT to ruff as `ruff fmt` supports isort and black rules (and is faster). The main reason I haven't is the different exclude lists would be annoying to port right now and we have all the open UFMT PRs.